app/uk/gov/hmrc/tradergoodsprofilesdatastore/models/response/ProfileResponse.scala

```diff
@@ -24,7 +24,8 @@ case class ProfileResponse(
   actorId: String,
   ukimsNumber: String,
   nirmsNumber: Option[String],
-  niphlNumber: Option[String]
+  niphlNumber: Option[String],
+  eoriChanged: Boolean
```

Comment:
Existing records does not have this field in db. For those this will fail :( 
We will have to change this to an Option
`eoriChanged: Option[Boolean]`